content/competitions/hackathon-2020.html

```diff
@@ -17,27 +17,66 @@ <h1 class="title is-3"><a href="https://www.facebook.com/events/2858423371101933
         <h2 class="subtitle is-4">5pm Friday 16th October &ndash; 8pm Sunday 18th October</h2>
 
         <div class="content">
-            <h4>
-                <a href="https://link.uqcs.org/hackathon">Get your tickets now!</a>
-            </h4>
+            <p>
+                <a href="https://link.uqcs.org/hackathon" class="button is-link">Get your tickets now!</a>
+                <a href="https://www.facebook.com/events/2858423371101933/" class="button">Facebook event</a>
+            </p>
 
             <p>Howdy, y'all! We're happy to announce our biggest event of the year, the UQCS Hackathon, will be on
-                October 16th to 18th.</p>
+                <b>October 16th to 18th</b> at Room 301 of the Advanced Engineering Building.</p>
 
-            <p>If you aren't familiar with how the UQCS Hackathon works, it's pretty simple:</p>
+                <p>
+                    Form a team on Friday night, then spend two days building something cool—a game, a website, hardware, 
+                    the sky's the limit. On Sunday, teams will present what they've done, with $2500 in prizes up for grabs! 
+                    And don't worry about food for the weekend; we'll be supplying breakfast, lunch and dinner each day.
+                </p>
+
+                <h3>How to Hackathon</h3>
+            <p>
+                <b>How to Hackathon with Alexander Blyth</b> is 
+                on Tuesday October 13th (<a href="https://www.facebook.com/events/668866880708703/">Facebook event</a>). 
+                if you've never been to a Hackathon before, this will walk you through what
+                    to do at your first Hackathon and how to make the most of the weekend.
+                Even if you've been to a hackathon before, come along for some hints and tips.
+                It's pretty simple:</p>
             <ul>
-                <li>Form your team
+                <li>Form your team</li>
                 <li>Build something cool</li>
                 <li>Eat Free Food</li>
                 <li>Present what you have made</li>
                 <li>Win prizes</li>
             </ul>
+
+            <h3>Projects</h3>
+            <p>
+                In the past people have built websites, desktop apps, games, hardware, and even robots. 
+                Some past projects are on our showcase page.
+                Still not sure what to do? Check out the ideas at the link below. 
+                If you have your own amazing idea, you can also add it and find teammates to work with!
+            </p>
+            <p>
+                <a class="button is-info" href="https://uqcs.org/showcase/">UQCS Showcase</a>
```

Comment:
I feel like this link should be relative

Reply:
Sure.